api/users/users-router.js

```diff
@@ -0,0 +1,42 @@
+const express = require(`express`)
+const Users = require(`./users-model`)
+const {validateLogin, validateRegister, validateUserId} = require(`../middleware`)
+
+const router = express.Router()
+
+router.get(`/users`, (req, res, next) => {
+   Users.find()
+      .then(user => {
+      res.status(200).json(user)
+      })
+      .catch(next)
```

Comment:
Nit: I know that the course you're following is teaching the catch error at the end of everything. Personally I think having the catch error sent within the response is better bc then you can add a message to it to be able to debug the code better later on. 

For example: this call would be 
`
router.get(`/users`, (req, res, next) => {
   Users.find()
      .then(user => {
      res.status(200).json(user)
      })
      .catch(err => {
          res.status(500).json({message: "Could not get users", error: err.message})
       }) 
`